README.md

```diff
@@ -19,10 +19,66 @@ If you do not have `golangci-lint` installed, review the `golangci-lint` [instal
 
 [golangci-lint-install]: https://golangci-lint.run/welcome/install/
 
+### Using with golangci-lint v2 Plugin
+
+Kube API Linter can be used as a plugin with `golangci-lint` v2. This setup allows integrating Kube API Linter alongside other static analysis tools within the same linter workflow.
+
+More information about golangci-lint plugins can be found in the [golangci-lint plugin documentation][golangci-lint-plugin-docs].
+
+#### Plugin Setup
+
+First, build the plugin `.so` file:
+
+```shell
+go build -buildmode=plugin -o ./bin/kube-api-linter.so sigs.k8s.io/kube-api-linter/pkg/plugin
+```
+
+This will generate a `kube-api-linter.so` plugin in the `bin` directory.
+
+#### Configuration
+
+Update your `.golangci.yml` with the following snippet:
+
+```yaml
+version: "2"
+
+linters:
+  enable:
+    - kubeapilinter
+    ...
+
+  settings:
+    revive:
+      rules:
+        - name: comment-spacings
+        - name: import-shadowing
+    custom:
+      kubeapilinter:
+        path: "./bin/kube-api-linter.so"
+        description: "Kube API Linter plugin"
+        original-url: "sigs.k8s.io/kube-api-linter"
+        settings:
+          linters: {}
+          lintersConfig: {}
+
+exclusions:
+  rules:
+    # Only run kubeapilinter inside the 'api/' directory
+    # Prevents running API-specific checks on unrelated packages like controllers or tests
+    - path-except: "^api/"
+      linters:
+        - kubeapilinter
```

Comment:
Is the formatting off for the exclusions stanza here? It looks like it needs to be linters.exclusions when I was looking at integrating this linter in my repository that uses the 2.x golangci-lint version.

Reply:
Oh I get your point, yes, you're correct, this whole block from exclusions down needs indenting 2 spaces